lib/themes/dosomething/paraneue_dosomething/js/utilities/ApiClient.js

```diff
@@ -36,6 +43,37 @@ class ApiClient {
   }
 
   /**
+   * Send a DELETE request to the given path URI.
+   *
+   * @param  {String} path
+   * @return {Object}
+   */
+  delete(path) {
+    const url = `${this.url}${path}`;
+
+    // @TODO: see todo in post() method!
+    console.log('%c ApiClient DELETE Request: \n' + url,
```

Comment:
Yep, I think this is a good idea to get in the habit of since there's no downside and can be massively useful for debugging (with useful messages, obvs, not like `console.log('💩')`).